src/modules/keyboardmanager/dll/KeyboardEventHandlers.cpp

```diff
@@ -260,6 +260,11 @@ namespace KeyboardEventHandlers
                     }
 
                     it.second.isShortcutInvoked = true;
+                    // If app specific shortcut is invoked, store the target application
+                    if (activatedApp != L"")
```

Comment:
[nit] Can we please use some constant variable rather than a direct comparison with an empty string?